tests/unit/test_store_handler.py

```diff
@@ -0,0 +1,141 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+""" Test Store handler features.  """
+
+import os
+import unittest
+
+from libcodechecker.server.api import store_handler
+from libcodechecker.analyze import plist_parser
+
+from codeCheckerDBAccess_v6 import ttypes
+
+
+class StoreHandler(unittest.TestCase):
+    """
+    Test Store handler features.
+    """
+
+    @classmethod
+    def setup_class(cls):
+        # Already generated plist files for the tests.
+        cls.__this_dir = os.path.dirname(__file__)
+        cls.__plist_test_files = os.path.join(
+            cls.__this_dir, 'plist_test_files')
+
+    def test_collect_path_events(self):
+        """
+        Test path event collect before store.
+        """
+        self.maxDiff = None
```

Comment:
Do not use camel cased variable names in python. Also check `Codacy` and try to fix the warnings.